argo-cd-apps/base/host/smee/kustomization.yaml

```diff
@@ -1,6 +1,3 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
why not deleting this kustomization file and parent folder argo-cd-apps/base/host/smee